lib/util/stream_utils.js

```diff
@@ -1445,3 +1469,44 @@ shaka.util.StreamUtils.DecodingAttributes = {
   POWER: 'powerEfficient',
   BANDWIDTH: 'bandwidth',
 };
+
+/**
+ * @private {!Set.<string>}
+ */
+shaka.util.StreamUtils.supportedImageMimeTypes = new Set()
```

Comment:
Instead of two `{Set.<string>}` instances, I would recommend a single `{Map.<string, boolean>}`.  Also, please end private variable names with an underscore.

```js
shaka.util.StreamUtils.supportedImageMimeTypes_ = new Map([
  ['image/svg+xml', true],
  ['image/png', true],
  ['image/jpeg', true],
  ['image/jpg', true],
]);
```

Reply:
Done!